app/components/GithubActionsScheduledJobs.tsx

```diff
@@ -0,0 +1,64 @@
+import Image from "next/image";
+
+function GithubActionsJob({
+  repo,
+  workflowName,
+}: {
+  repo: string;
+  workflowName: string;
+}) {
+  return (
+    <div>
+      <a
+        href={`https://github.com/ISISComputingGroup/${repo}/actions/workflows/${workflowName}`}
+        className={
+          "flex-row flex items-center content-center h-10 w-full text-center rounded-lg bg-gray-800 border-2 border-black hover:border-white"
+        }
+      >
+        <h2 className={"text-center text-lg w-1/2"}>{repo}: </h2>
```

Comment:
nit, but we might not even really need the trailing colon here, I think it'd look fine without...